src/Http/Middleware/HttpsEnforcerMiddleware.php

```diff
@@ -53,6 +54,7 @@ class HttpsEnforcerMiddleware implements MiddlewareInterface
         'statusCode' => 301,
         'headers' => [],
         'disableOnDebug' => true,
+        'trustedProxies' => null,
```

Comment:
I think array should probably default to `[]`, no?

Reply:
I would be careful with that as ServerRequest::setTrustedProxies() happily accepts empty array too.

Reply:
So empty means all? That's unfortunate.

Reply:
yes: https://github.com/cakephp/cakephp/pull/16855/files#diff-93bf558bfa319a9d7005a51bc3fff702371b069e35abe89cb94111d906133006R80